projects/core/directives/dropdown/dropdown-context.directive.ts

```diff
@@ -22,14 +25,24 @@ function activeZoneFilter(this: TuiDropdownContextDirective, target: Element): b
         tuiAsDriver(TuiDropdownDriver),
         tuiAsRectAccessor(TuiDropdownContextDirective),
     ],
+    host: {
+        '[style.user-select]': '"none"',
+        '[style.-webkit-touch-callout]': '"none"',
+        '[style.-webkit-user-select]': '"none"',
+    },
 })
 export class TuiDropdownContextDirective extends TuiRectAccessor {
     private readonly driver = inject(TuiDropdownDriver);
     private currentRect = EMPTY_CLIENT_RECT;
+    private longTapTimeout?: any;
```

Comment:
why `any`?

Reply:
it conflicts with Node.js setTimeout, that returns `Timeout` object insted of `number`

Reply:
![Uploading image.png…]()

Reply:
fixed